home/models.py

```diff
@@ -1222,3 +1214,35 @@ def clean(self):
 
     def __str__(self):
         return f'{self.locale}'
+
+
+def get_all_renditions_urls(image):
```

Comment:
My preference would be to move these to `utils.py` or something like that to keep things organized.

Reply:
Sounds good to me, however, there is already a `home.utils` module with sub-modules. Ideally, I would like to merge all the sub-modules into a single `home.utils` module, and then add these functions as well, but is that too much change?

An alternative would be to add an '__init__.py' in 'home/utils' and add the functions there - what do you think?

Reply:
The first option sounds reasonable but one thing to consider in this approach is how big we want a module to be. If you think it's more manageable to combine utils then please go ahead ( my preference would be to do it in a separate PR to keep this PR relevant & to the point).

Another option would be to keep things in their module and import them in `home/utils/__init__.py` so other modules can import things from `home.utils` directly instead of each individual module.

Reply:
I'd to like to move towards a single utils module because there isn't enough code there to justify the existence of sub-modules, IMO. With that in mind, I have moved the functions from `home.models` to `home.utils`. I will create a separate PR to merge the sub-modules.